src/item.cpp

```diff
@@ -1484,7 +1484,7 @@ static const double hits_by_accuracy[41] = {
     9993, 9997, 9998, 9999, 10000 // 16 to 20
 };
 
-double item::effective_dps( const Character &guy, monster &mon ) const
```

Comment:
These changes should also be a separate PR; they're correct whether or not we do the NPC AI rework.